src/muya/lib/contentState/imageCtrl.js

```diff
@@ -125,6 +125,9 @@ const imageCtrl = ContentState => {
       start: { key, offset: start },
       end: { key, offset: start }
     }
+    setTimeout(() => {
```

Comment:
It maybe no need to use setTimeout, just put `this.muya.dispatchChange` call after `this.singleRender(block)`?

Reply:
Like this？
```js
return this.singleRender(block), this.muya.dispatchChange()
```
or
```js
this.singleRender(block)
return this.muya.dispatchChange()
```

Reply:
I prefer the second code style. Do you test it?

Reply:
Yes, I did. It works.